Exclude menus from lists (13. Content Structure - Visually Apparent Lists) (#375)
atbcb/ICTTestingBaseline: _baselines/ChangeLog3.md

| 10.1 Form Names | Instruction 2: separated to 2 test instructions, added SC 1.1.1. New instruction 2 maps to SC 1.1.1 and new instruction 3 maps to SC 4.1.2 |
| 10.1, 10.2, 10.5 | Identify Content: Added "that do not have <code>visibility:hidden</code> or <code>display:none</code>" after "Find all form components". | 
| 10.5 Form has a Label | Test Procedure name changed to "Form has a Visible Label" and instruction 1 edited to include "has visible label(s) or instructions while the form component has focus." |

> | 13. Content Structure | Limitations, Assumptions, Exceptions: Added "While an unordered list structure is often used to create navigation menus, menus can be created using other techniques such as `<nav>`. The test for Visually Apparent Lists does not include navigation menus." |

Author: ```suggestion
| 13. Content Structure | Limitations, Assumptions, Exceptions: Added "A test for Visually Apparent Lists should not include navigation menus. While programmatic lists are often used to create navigation menus, menus may also be created using other techniques." |
```
